source/jormungandr/jormungandr/scenarios/simple.py

```diff
@@ -119,7 +119,15 @@ def line_reports(self, request, instance):
         req = request_pb2.Request()
         req.requested_api = type_pb2.line_reports
         req.line_reports.depth = request['depth']
-        req.line_reports.filter = request['filter']
+
+        if isinstance(request['filter'], str):
+            req.line_reports.filter = request['filter']
+        else:
+            req.line_reports.object_type = request_pb2.LineReportsRequest.Type.Value(
+                request['filter']['object_type']
+            )
+            req.line_reports.object_id = request['filter']['object_id']
```

Comment:
Maybe check that that `request['filter']` is a dict that have the attributes 'object_type' and 'object_id'.

Reply:
fixed